### DISCUSSION TITLE
Should we use crowdin?

### DISCUSSION THREAD

**Discussion by armandfardeau** [OpenSourcePolitics/osp-app-k8s#49] [Ideas]
Should we use crowdin for managing locales?

**Comment by Quentinchampenois** [comment]
I think it can be interesting in the case of customizations or if we need to define new locales.
However, locales are currently managed using Crowdin as mentioned [here](https://docs.decidim.org/en/develop/managing_translations_i18n/#_the_workflow), could it be a duplicate ?

The idea is great to me and it can be useful

**Reply by armandfardeau** [reply]
I think it might be a duplicate, let's enforce locales and it should be alright
